src/platforms/ios/iosgleanbridge.mm

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "iosgleanbridge.h"
+#include "leakdetector.h"
+#include "Mozilla_VPN-Swift.h"
+
+namespace {
+
+// Our Swift singleton.
+IOSGleanBridgeImpl* impl = nullptr;
```

Comment:
can this be a member variable of `IOSGleanBridge` instead of a global var?

Reply:
I think that is not an issue. I was just following the pattern on IOSController though.